src/internal.rs

```diff
@@ -14,7 +16,15 @@ pub unsafe fn copy_to_c_buffer(mut src: String, dest: *mut c_char) {
     ptr::copy_nonoverlapping(src_c.as_ptr(), dest, src_c_length);
 }
 
+/// Enables native paths
+pub fn path_init() {
```

Comment:
```suggestion
fn path_init() {
```
Unless I am missing something, there should be no reason anymore for this to be public